.github/workflows/ci.yml

```diff
@@ -28,10 +28,9 @@ jobs:
         uses: actions/checkout@61b9e3751b92087fd0b06925ba6dd6314e06f089
 
       - name: Rust toolchain
-        uses: actions-rs/toolchain@16499b5e05bf2e26879000db0c1d13f7e13fa3af
+        uses: dtolnay/rust-toolchain@1.73.0
```

Comment:
Can we use the hash like the other actions

Reply:
not sure? https://github.com/marketplace/actions/rustup-toolchain-install

Reply:
Maybe something like this?

```yaml
     # Installs the stable toolchain which preceded the most recent one by
     # the specified number of minor versions.
  - uses: dtolnay/rust-toolchain@439cf607258077187679211f12aa6f19af4a0af7
    with:
      toolchain: stable minus 8 releases
```

Reply:
Done